nucypher/cli/literature.py

```diff
@@ -363,7 +375,7 @@
 #
 
 
-CONFIRM_URSULA_IPV4_ADDRESS = "Is this the public-facing IPv4 address ({rest_host}) you want to use for Ursula?"
+CONFIRM_URSULA_IPV4_ADDRESS = "Detected IPv4 address ({rest_host}) - Is this is the public-facing address of Ursula?"
```

Comment:
```suggestion
CONFIRM_URSULA_IPV4_ADDRESS = "Detected IPv4 address ({rest_host}) - Is this the public-facing address of Ursula?"
```

Reply:
Unresolving since the text hasn't been updated...

Reply:
Whoops! I accidentally forced pushed over this branch after merging suggestions without pulling.